src/plugin.yml

```diff
@@ -231,35 +232,32 @@ permissions:
     children:
       bending.ability.Immobilize: true
   bending.air.passive:
-    default:  false
+    default: false
     description: Grants access to all air passives.
     children:
       bending.ability.AirAgility: true
       bending.ability.AirSaturation: true
       bending.ability.GracefulDescent: true
   bending.chi.passive:
-    default:  false
+    default: false
     description: Grants access to all chi passives.
     children:
       bending.ability.Acrobatics: true
       bending.ability.ChiAgility: true
       bending.ability.ChiSaturation: true
   bending.earth.passive:
-    default:  false
+    default: false
     description: Grants access to all earth passives.
     children:
       bending.ability.DensityShift: true
       bending.ability.SandAgility: true
   bending.fire.passive:
-    default:  false
+    default: false
     description: Grants access to all fire passives.
     children:
   bending.water.passive:
-    default:  false
+    default: false
     description: Grants access to all water passives.
     children:
       bending.ability.FastSwim: true
       bending.ability.HydroSink: true
-  bending.donor:
-    default: false
-    description: Grants the Donor tag.
```

Comment:
The usage of this permission should also be removed from the Who command

Reply:
Didn't even know it had it. Will do 👍